config/samples/go_example_loadtest.yaml

```diff
@@ -29,6 +29,9 @@ spec:
     run:
       command:
       - /src/workspace/bin/worker
+  driver:
```

Comment:
You should probably not change this as part of this PR, but only after the next release is deployed.